src/shared/components/query/styles.module.scss

```diff
@@ -219,7 +225,7 @@ div.SelectedStudiesWindow {
       color: #043852;
       border-bottom: 1px solid rgb(222, 233, 241);
       border-top: 1px solid rgb(243, 248, 251);
-      padding: 8px 10px;
+      padding: 8px 16px 8px 10px;

```

Comment:
This slightly increases right padding for the cancer type list item (the number next to cancer type). Those numbers are partially covered by the scrollbar in IE, Firefox, and Safari. This makes them fully visible.